docs/architecture_overview.md

```diff
@@ -71,20 +71,20 @@ There are three main abstractions for Mephisto that comprise differences in runn
 
 There's also the abstraction of the `MephistoDB`, which defines the data operations that Mephisto requires to operate properly. If a specific setup requires specialized data handling, any class implementing this interface could stand in for it.
 
 ## Blueprints
-The [blueprints](https://github.com/facebookresearch/Mephisto/tree/master/mephisto/server/blueprints) contain all of the related code required to set up a task run. Blueprints may follow a hierarchical structure, in that general functionality can be written into abstract blueprints (which are powerful but perhaps hard to configure) and then downstream blueprints may have more configuration control but less breadth. Much of this can actually require significant overhead, so we've created additional abstract classes that a blueprint must link an implementation for. These are listed below:
+The [blueprints](https://github.com/facebookresearch/Mephisto/tree/master/mephisto/server/blueprints) contain all of the related code required to set up a task run. Blueprints may follow a hierarchical structure, in that general functionality can be written into abstract blueprints (which are powerful but perhaps hard to configure) and then downstream blueprints may have more configuration control but less breadth. Much of this can actually require significant overhead, so we've created additional abstract classes that a blueprint must link to an implementation for. These are listed below:
 ### BlueprintArgs
 These define the specific arguments for configuring a blueprint. Some base arguments relevant to all blueprints are provided already, and classes that override `BlueprintArgs` can add anything else they want.
 ### SharedTaskState
 Some tasks require additional state that cannot be considered static configuration, such as validation functions for accurate completion of a task, or a counter for the number of tasks completed in a run so far, or any other python state that multiple different assignments may be interested in accessing or altering. This class is for the edge cases that need something more than static configuration, and is created as a singleton shared between all tasks in a `TaskRun`.
 ### TaskBuilder
-All tasks need to have some kind of frontend interface for workers to interact with. The `TaskBuilder` defines hooks for points in setup where this frontend is compiled and built (or located and confirmed alive), and implementations for a specific `Blueprint` should use this part of the lifecycle to set up these things.
+All tasks need to have some kind of frontend interface for workers to interact with. The `TaskBuilder` defines hooks for points in a setup where this frontend is compiled and built (or located and confirmed alive), and implementations for a specific `Blueprint` should use this part of the lifecycle to set up these things.
```

Comment:
"in" in this location was really trying to convey something more like "during" the setup of a run - trying to describe that every `TaskRun` will need to point people at something (likely hosted on the server deployed by the architect), the `TaskBuilder` is responsible for actually creating that content for the server to host.

Reply:
oops - will revert! could also change to during